packages/test/test-driver-definitions/src/interfaces.ts

```diff
@@ -10,26 +10,32 @@ import {
 } from "@fluidframework/driver-definitions";
 
 /**
+ * Types of test drivers.
```

Comment:
https://dev.azure.com/fluidframework/public/_build/results?buildId=213129&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=bfe27d5f-9219-57c0-0d67-1b2946a32921&l=167

--

Added somewhat "obvious" description as `api-extractor` would throw errors. @alexvy86 @Josmithr  Your thoughts?

Reply:
Looks ok to me, thanks!

Reply:
Nit: it would probably be useful to also document what the string values mean / do. Noting that (presumably) `tinylicious` and `t9s` are the same things, etc.